tests/php/Forms/LookupFieldTest.php

```diff
@@ -124,4 +126,21 @@ public function testWithMultiDimensionalSource()
             trim($f->Field()->getValue())
         );
     }
+
+    public function testAtomicValueHandlingFromDataObject()
+    {
+        $member = $this->objFromFixture(Member::class, 'member1');
+
+        /** @var LookupField $testField */
```

Comment:
Add an `$this->assertInstanceOf` as well

Reply:
Donzo.

Reply:
Donzo